bindings/python/python/opendal/__init__.pyi

```diff
@@ -40,6 +40,9 @@ class AsyncOperator:
     async def delete(self, path: str): ...
     async def list(self, path: str) -> AsyncIterable[Entry]: ...
     async def scan(self, path: str) -> AsyncIterable[Entry]: ...
+    async def presign_stat(self, path: str, expire: int) -> PresignedRequest: ...
```

Comment:
Does python has native duration type？I don't like the idea of using `int` instead of `Duration`.

Reply:
Python has `datetime.timedelta`. But pyo3's binding to it is only available on `non-Py_LIMITED_API`. It's conflict with `abi3-py37` feature.

Reply:
Hi, @messense, `timedelta` has been added since python 2.3. All existing python version should support `timedelta`. Why `&PyDelta` is marked as a `non-Py_LIMITED_API`?

Reply:
Ok, let's use `int`. Please make it clear that it's `seconds`.

Reply:
> Please make it clear that it's seconds.

done in new commit